### ISSUE TITLE
Error status if just one of trackers is unreachable

### ISSUE THREAD

**Issue by AlessandroBerrone** [jfurrow/flood#323]
Not really a bug, just a "wrong information" given to the user.

If just one of the trackers is unreachable or miswritten, flood flags the torrent with the ERROR status. Funny fact is thah the torrent begins and ends the download without any problem, because it successfully connect to the other trackers.

So there is just to fix the "error" flagging condition, because at the moment almost every torrent gets flagged as "error" even if it downloads fine.

![opera_2017-04-24_18-17-54](https://cloud.githubusercontent.com/assets/9112120/25347467/523880ae-291b-11e7-94bf-8c791f072537.png)
![opera_2017-04-24_18-18-05](https://cloud.githubusercontent.com/assets/9112120/25347468/5253b68a-291b-11e7-9a82-5d6e4c24918c.png)

**Comment by jfurrow** [comment]
Yeah, this is a good call. Thanks for the issue! I'm not sure what the best way to handle this is, but I'll come up with something. It would be awesome if someone could investigate how other UIs determine an error state (specifically ruTorrent), but I'll tackle that if needed.
